doc/source/names_wordlist.txt

```diff
@@ -1,1652 +1,1987 @@
-Critchley
-Villanova
```

Comment:
do we need this list at all? (as you are now getting it from the authors)?

Reply:
Yeah the list is build when spellcheck is called. I have updated the spellcheck function located in make.py to call the function get_name_wordlist() which is located in scripts/announce.py

Reply:
ok, so let's delete the list then.

Reply:
Unfortunately, we still need the list, the only reason why we do need it is due to the fact that there are names(around 5-7) in the file `release.rst` that are not author names (or at least they are different so they don't get added when we run the function `get_names_wordlist`. 

If these names are not added somewhere then the build will always fail. If you are worried about having a file with all the names of every contributor I could just delete most of them and add only those in `release.rst` since the file will be updated with all the names.

Perhaps this file could be good to have just in case people need to add their own names (like they use a different one or something and it gets flagged as a spelling error like those in `release.rst`), so with this file people would know that they have to add those names in there and not in the `spelling_wordlist.txt`, what do you think?

Reply:
> I could just delete most of them 

Yes, I would only keep those that are not generated by the script (so the ones that are needed to get the script running)

Reply:
> Yes, I would only keep those that are not generated by the script (so the ones that are needed to get the script running)

Actually, another option is to have them all in there, and have the script update it on demand (so if the release manager adds the names to the notes, the scripts needs to be run). That would save the extra step of creating it on each doc build.